scripts/commands/stopgarrison.lua

```diff
@@ -0,0 +1,17 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
For ease of life, I'd recommend refactoring this so that it can be used like this:
```
!garrison start
!garrison stop
```

And presumably, like with any version of something, it will probably break and players will get left in a funky state. Maybe make this act on the cursor target so you can bypass if someone has lockout; so a GM can do `!garrison start` on someone who's had their garrison break

Reply:
Good call. I was thinking of doing something similar (also to do something like !garrison timeleft etc.. for debugging) so I will add these too.

Reply:
Let's do this as a follow up MR so we can land the big bulk first. Will work on it today.